apps/web/src/pages/Applications/ApplicationProfilePage/components/PersonalInformation/PersonalInformation.tsx

```diff
@@ -0,0 +1,116 @@
+import React from "react";
+import { useIntl } from "react-intl";
+import { SubmitHandler } from "react-hook-form";
+import { UserIcon } from "@heroicons/react/24/outline";
+
+import { ToggleSection, Well } from "@gc-digital-talent/ui";
+import { toast } from "@gc-digital-talent/toast";
+import { BasicForm } from "@gc-digital-talent/forms";
+
+import profileMessages from "~/messages/profileMessages";
+import {
+  hasAllEmptyFields,
+  hasEmptyRequiredFields,
+} from "~/validators/profile/about";
+
+import { getLabels, formValuesToSubmitData, dataToFormValues } from "./utils";
+import { FormValues } from "./types";
+import { SectionProps } from "../../types";
+import { getSectionIcon, getSectionTitle } from "../../utils";
+import SectionTrigger from "../SectionTrigger";
+import FormActions from "../FormActions";
+import NullDisplay from "./NullDisplay";
+import Display from "./Display";
+import FormFields from "./FormFields";
+
+const PersonalInformation = ({ user, onUpdate, isUpdating }: SectionProps) => {
+  const intl = useIntl();
+  const labels = getLabels(intl);
+  const isNull = hasAllEmptyFields(user);
+  const [isEditing, setIsEditing] = React.useState<boolean>(false);
+  const title = getSectionTitle("personal");
+  const icon = getSectionIcon({
+    isEditing,
+    error: false,
+    completed: false,
+    fallback: UserIcon,
+  });
+
+  const handleSubmit: SubmitHandler<FormValues> = async (formValues) => {
+    await onUpdate(user.id, formValuesToSubmitData(formValues, user))
+      .then(() => {
+        toast.success(
+          intl.formatMessage({
+            defaultMessage:
+              "Personal and contact information updated successfully!",
+            id: "J+MAUg",
+            description:
+              "Message displayed when a user successfully updates their personal and contact information.",
+          }),
+        );
+        setIsEditing(false);
+      })
+      .catch(() => {
+        toast.error(intl.formatMessage(profileMessages.updatingFailed));
+      });
+  };
+
+  return (
+    <ToggleSection.Root
+      id="personal-section"
+      open={isEditing}
+      onOpenChange={setIsEditing}
+    >
+      <ToggleSection.Header
```

Comment:
The alignment looks off on these.
![image](https://user-images.githubusercontent.com/8978655/233141903-5fa198ec-95ea-4bbb-846a-dd7ebbade694.png)

Reply:
Fixed in [change toggle align to be in center](https://github.com/GCTC-NTGC/gc-digital-talent/pull/6296/commits/62e2d9fbb423a854ff49153e63a09e92bddeff54)

Reply:
Very nice.